src/components/gameOverWindow/GameOver.component.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import * as S from './styled.component';
+import BestScore from '../bestScore/BestScore.component';
+
+const GameOver = ({ parameters, updateIsGameOver, saveItem }) => {
+  function newGame(event) {
+    event.keyCode === 32 ? updateIsGameOver(false) : null;
+    parameters.score = 0;
+    window.removeEventListener('keydown', newGame);
+  }
+  window.addEventListener('keydown', newGame);
```

Comment:
use `[useEffect](https://ru.reactjs.org/docs/hooks-effect.html)` hook for side effects